metrics/metrics.go

```diff
@@ -99,6 +100,12 @@ func NewShadowsocksMetrics(ipCountryDB *geoip2.Reader) ShadowsocksMetrics {
 				Name:      "data_bytes",
 				Help:      "Bytes transferred by the proxy",
 			}, []string{"dir", "proto", "location", "status", "access_key"}),
+		tcpProbes: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: "shadowsocks",
+			Name:      "tcp_probes",
+			Buckets:   []float64{49, 50, 51},
```

Comment:
Could you add 0 and 48, so we know the counts for 0, [1, 48], 49, 50, 51, [52, Inf]?

Is there any other interesting number we want to capture?